### PR TITLE
Chore cleanups

### PR DESCRIPTION
**Pull request by drahnr**

### CODE REVIEW

**Comment by drahnr** [comment]
Should be merged before / is part of #710

**Comment by drahnr** [comment]
Is there anything to do to get the chore merged? :)

File: Cargo.toml [hunk 1]
```diff
@@ -77,7 +77,7 @@ tokio-serde-bincode = "0.1"
 tower = "0.1"
 tokio-tcp = "0.1"
 tokio-timer = "0.2"
-toml = "0.4"
+toml = "0.5"
```
Comment (glandium) [inline Cargo.toml:80]: This was done in b6cede2

**Comment by glandium** [comment]
Many of these cleanups were merged in 3b340b3. I'm keeping this opened to keep an eye on the remainder.

File: Cargo.toml (hunk 1 above)
Comment (drahnr) [reply Cargo.toml:80]: Well, this PR was opened 8 months ago.

**Comment by drahnr** [comment]
@glandium I'd love to have a chat about the future of `sccache` development, since a lot things have stalled and it seemingly became dormant. Also see #876

**Comment by drahnr** [comment]
This PR could probably be closed, it's more chore to rebase it than to re-create the changes.

**Comment by glandium** [comment]
I'm first getting a picture of the current state and will followup in that bug after I'm done going through PRs and issues.

**Comment by drahnr** [comment]
I've been plowing through the futures 0.1 to futures 0.3 code which is almost complete in https://github.com/paritytech/sccache and a few more things lined up. So from there it will be significantly easier to refactor and/or write new code. Just something to keep in mind.

**Comment by glandium** [comment]
Applied most of the remainder in bcd831269647dee56182d01738faf373daba3e67

**Pull request closed without merging**